lib/engines/content_object_store/app/models/concerns/content_object_store/hasAuthors.rb

```diff
@@ -1,6 +1,7 @@
 module ContentObjectStore
   module HasAuthors
     extend ActiveSupport::Concern
+    include ContentObjectStore::HasCreator
```

Comment:
@pezholio I'm popping the `HasCreator` into `HasAuthors` as it directly depends on the Edition already having authors, but maybe it should be included on the Edition itself?